content/zh-cn/docs/concepts/architecture/controller.md

```diff
@@ -218,6 +221,8 @@ allow for that.
 作为设计原则之一，Kubernetes 使用了很多控制器，每个控制器管理集群状态的一个特定方面。
 最常见的一个特定的控制器使用一种类型的资源作为它的期望状态，
 控制器管理控制另外一种类型的资源向它的期望状态演化。
+例如，Job 的控制器跟踪 Job 对象（以发现新的任务）和 Pod 对象（以运行Job，然后查看任务何时完成）。
```

Comment:
Missed the space between English and Mandarin words.

Reply:
> Missed the space between English and Mandarin words.

Thank you for your thoughtfulness and responsibility